src/Inventory.zig

```diff
@@ -329,7 +329,21 @@ pub const Sync = struct { // MARK: Sync
 
 					inventory.inv.loadFromZon(inventoryZon);
 				},
-				.other => {},
+				.other => {
```

Comment:
You should add a new source type for that, otherwise it could get confused with other inventories, like e.g. the crafting recipe inventories.

And you wouldn't want your last hand items to appear when you open a workbench either.